15:54 < _rene_> (right now it seems apparmor rules allowing /bin/dash don't 
                allow /usr/bin/dash (where /bin -> (isr/bin due to usrmerge)
15:55 < _rene_> need to verify that in a clean no-usrmerge-VM though
15:56  * bigon says something about selinux policy supporting usrmerge
15:56 < bigon> :p
15:58 < debfx> _rene_: that's right. those rules need to be changed to 
               /{usr/,}bin/dash
16:04 < _rene_> debfx: thanks for confirming. will change that (and grep, sed, 
                ..)

diff --git a/sysui/desktop/apparmor/program.senddoc b/sysui/desktop/apparmor/program.senddoc
index 42c2740f7012..c08e78390a4f 100644
--- a/sysui/desktop/apparmor/program.senddoc
+++ b/sysui/desktop/apparmor/program.senddoc
@@ -17,14 +19,14 @@ profile libreoffice-senddoc INSTDIR-program/senddoc
 
   owner /tmp/lu**       rw,    #makes files like luRRRRR.tmp/lubRRRR.tmp where R is random
                                #Note, usually it's lub or luc, don't know why.
-  /bin/sh               rmix,
-  /bin/dash             rmix,
-  /bin/bash             rmix,
-  /bin/sed              rmix,
+  /{usr/,}bin/sh        rmix,
+  /{usr/,}bin/bash      rmix,
+  /{usr/,}bin/dash      rmix,
+  /{usr/,}bin/sed       rmix,
   /usr/bin/dirname      rmix,
   /usr/bin/basename     rmix,
-  /bin/grep             rmix,
-  /bin/uname            rmix,
+  /{usr/,}bin/grep      rmix,
+  /{usr/,}bin/uname     rmix,
   /usr/bin/xdg-open     Cxr -> sanitized_helper,
   /usr/bin/xdg-email    Cxr -> sanitized_helper,
   /dev/null             rw,
diff --git a/sysui/desktop/apparmor/program.soffice.bin b/sysui/desktop/apparmor/program.soffice.bin
index b4baeab8f5ee..426a04875e13 100644
--- a/sysui/desktop/apparmor/program.soffice.bin
+++ b/sysui/desktop/apparmor/program.soffice.bin
@@ -105,9 +107,9 @@ profile libreoffice-soffice INSTDIR-program/soffice.bin {
   # bluetooth send to
   network bluetooth,
 
-  /bin/sh                               rmix,
-  /bin/bash                             rmix,
-  /bin/dash                             rmix,
+  /{usr/,}bin/sh                        rmix,
+  /{usr/,}bin/bash                      rmix,
+  /{usr/,}bin/dash                      rmix,
   /usr/bin/bluetooth-sendto             rmPUx,
   /usr/bin/lpr                          rmPUx,
   /usr/bin/paperconf                    rmix,
